pkgs/development/python-modules/tokenizers/default.nix

```diff
@@ -105,6 +107,11 @@ in rustPlatform.buildRustPackage rec {
     pipInstallPhase
   '';
 
+  disabledTests = [
```

Comment:
```suggestion
  preCheck = ''
    HOME=$TMPDIR
  '';
  disabledTests = [
```

```
/nix/store/yi9a1fafijn0nf5m8qkmnl955vi2mncm-python3-3.8.6/lib/python3.8/os.py:213: in makedirs
    makedirs(head, exist_ok=exist_ok)
/nix/store/yi9a1fafijn0nf5m8qkmnl955vi2mncm-python3-3.8.6/lib/python3.8/os.py:223: in makedirs
    mkdir(name, mode)
E   PermissionError: [Errno 13] Permission denied: '/homeless-shelter'
```

Reply:
Thanks, and sorry for the delay (busy week). Should be fixed now.